deepcell/model_zoo/panopticnet.py

```diff
@@ -76,26 +76,35 @@ def __merge_temporal_features(feature, mode='conv', feature_size=256,
             raise ValueError('Mode {} not supported. Please choose '
                              'from {}.'.format(mode, str(acceptable_modes)))
 
+    f_name = str(feature.name)[:2]
+
+    conv_name = 'conv_mtf_{}'.format(f_name)
+    bnorm_name = 'bnorm_mtf_{}'.format(f_name)
+    acti_name = 'acti_mtf_{}'.format(f_name)

```

Comment:
If these are only used inside one of the blocks, just define them in that block (maybe even just skip the var name and pass in the string to the layer directly).

Reply:
Done